prestoadmin/deploy.py

```diff
@@ -142,7 +143,11 @@ def deploy_node_properties(content, remote_dir):
     _LOGGER.info("Deploying node.properties configuration")
     name = "node.properties"
     node_file_path = (os.path.join(remote_dir, name))
-    secure_create_file(node_file_path, PRESTO_STANDALONE_USER_GROUP)
+    if not exists(node_file_path, use_sudo=True):
+        secure_create_file(node_file_path, PRESTO_STANDALONE_USER_GROUP, mode=644)
+    else:
```

Comment:
we need to do this because the rpm doesn't set up the config files with correct ownership (see https://github.com/Teradata/presto/pull/358)